nx-X11/config/cf/X11.tmpl

```diff
@@ -625,23 +624,13 @@ XORGRELSTRING = XorgManVersionString
 #ifndef BuildXfixes
 #define BuildXfixes		YES
 #endif
-#ifndef BuildXfixesLibrary
-#define BuildXfixesLibrary	!BuildServersOnly
-#endif
```

Comment:
Btw, what’s the difference between `BuildFoo` and `BuildFooLibrary`? Does the first one define whether to build the `XFoo-Proto` part or not and the second one is relating to `libfoo`?